File: tls-sample/app/src/main/java/de/cotech/hw/tls/example/MyCustomApplication.java
package de.cotech.hw.tls.example;


import android.app.Application;

import de.cotech.hw.SecurityKeyManager;
import de.cotech.hw.SecurityKeyManagerConfig;

public class MyCustomApplication extends Application {

    public static final String TAG = "TLS";

    @Override
    public void onCreate() {
        super.onCreate();

        SecurityKeyManagerConfig config = new SecurityKeyManagerConfig.Builder()
                .setEnableDebugLogging(BuildConfig.DEBUG)
                .build();
        SecurityKeyManager.getInstance().init(this, config);
    }
}

File: jsch-sample/app/src/main/java/de/cotech/hw/ssh/sample/MyCustomApplication.kt
package de.cotech.hw.ssh.sample

import android.app.Application
import de.cotech.hw.SecurityKeyManager
import de.cotech.hw.SecurityKeyManagerConfig

class MyCustomApplication : Application() {

    companion object {
        const val TAG = "SSH"
    }

    override fun onCreate() {
        super.onCreate()

        val securityKeyManager = SecurityKeyManager.getInstance()
        val config = SecurityKeyManagerConfig.Builder()
            .setEnableDebugLogging(BuildConfig.DEBUG)
            .build()
        securityKeyManager.init(this, config)
    }
}
